src/Categories/Category/Instance/Properties/Setoids/Extensive.agda

```diff
@@ -3,6 +3,7 @@
 module Categories.Category.Instance.Properties.Setoids.Extensive where
 
 open import Level using (Level)
+open import Data.Empty renaming (⊥ to ⊥′)
```

Comment:
Use `Data.Empty.Polymorphic` instead and you won't have to `Lift` below

Reply:
Thanks, that makes things a little nice below